src/ucs/datastruct/ptr_array.h

```diff
@@ -57,29 +73,36 @@ typedef struct ucs_ptr_array {
 /**
  * Initialize the array.
  *
- * @param init_placeholder   Default placeholder value.
+ * @param [in] ptr_array          Pointer to a ptr array.
+ * @param [in] init_placeholder   Default placeholder value.
```

Comment:
`init_placeholder` can be removed as unneeded anymore?

Reply:
@dmitrygx, Well, it can be removed but changing the interface of the ptr_array at the point would mean changing the entire code... 
My recommendation is to remove it as part of a separate pull request.

Reply:
> My recommendation is to remove it as part of a separate pull request.

checking the code, we should update IB/RC and IB/UD and GTEST/UCS's test _datatype.cc

Reply:
OK, let me look into it - Please review the latest code.

Reply:
OK, it's not a lot of work indeed - Fixed.

@dmitrygx  - Please review the latest code.